meilisearch/index.py

```diff
@@ -575,24 +575,24 @@ def add_documents_ndjson(
 
     def add_documents_raw(
         self,
-        str_documents: str,
+        str_documents: bytes,
+        content_type: str,
         primary_key: Optional[str] = None,
-        content_type: Optional[str] = None,
```

Comment:
`content_type` is still needed for the HTTP header. I believe adding this back will fix the tests.

Reply:
`content_type` is still there – I just changed `Optional[str]` to `str` and moved it up with the other required fields. Do you want me to move it back down?

Reply:
Ah ok, I see it now. Yes, lets move it back to the and keep the optional otherwise this will be a breaking change since it isn't a name only parameter.

Reply:
Done.